src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_NAME = "supply";
+    private static final String BUY_NAME = "buy";
+    private static final String RESULT_NAME = "result";
+    private static final int CATEGORY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String PAIR_SEPARATOR = ":";
+    private static final String COMMA = ",";
+
+    private static String readDataFromFile(String fromFileName) {
+        StringBuilder readData = new StringBuilder();
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
Better not to use abbreviations in naming
```suggestion
        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
```